sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet/ssd_entry_point.py

```diff
@@ -260,44 +179,55 @@ def model_fn(model_dir):
     :param: model_dir The directory where model files are stored.
     :return: a model (in this case a Gluon network)
     """
+    import neomxnet  # noqa: F401
     net = gluon.SymbolBlock.imports(
-        '%s/model-symbol.json' % model_dir,
+        '%s/compiled-symbol.json' % model_dir,
         ['data'],
-        '%s/model-0000.params' % model_dir,
+        '%s/compiled-0000.params' % model_dir,
     )
```

Comment:
add `ctx=ctx` to `SymbolBlock.imports` to load the params into the appropriate context. Then we dont need the `reset_ctx` below

Reply:
done